playbooks/group_vars/benchmark_hosts/00-defaults.yml

```diff
@@ -13,77 +13,106 @@ fio_bench_list_default:
   - src: "fio_test_file.cfg.j2"
-    name: "fio_test_file_read_EBS"
+    name: "fio_file_randread_test_EBS"
     override: "{{ fio_test_file_read_16k_overrides | default({}) }}"
     blocksize: "16k"
     iodepth: 1
     rw: randread
     numjobs: 16
-    run_bench: "{{ ssd_bench | default(True) }}"
+    run_bench: "{{ ebs_file_bench | default(false) }}"
   - src: "fio_test_file.cfg.j2"
-    name: "fio_test_file_write_4k_iodepth8"
+    name: "fio_file_write_4k_test_OSA"
     override: "{{ fio_test_file_write_4k_overrides | default({}) }}"
     blocksize: "4k"
+    rw: write
     iodepth: 8
     numjobs: 1
-    run_bench: "{{ ssd_bench | default(True) }}"
+    run_bench: "{{ osa_file_bench | default(false) }}"
   - src: "fio_test_file.cfg.j2"
-    name: "fio_test_file_read_4k_iodepth8"
+    name: "fio_file_read_test_OSA"
     override: "{{ fio_test_file_read_4k_overrides | default({}) }}"
     blocksize: "4k"
+    rw: read
     iodepth: 8
     numjobs: 1
-    run_bench: "{{ ssd_bench | default(True) }}"
+    run_bench: "{{ osa_file_bench | default(false) }}"
   - src: "fio_direct_test.cfg.j2"
-    name: "fio_direct_write_test_RBD"
+    name: "fio_rbd_1M_write_test_EBS"
     override: "{{ fio_rbd_write_test_1024k_overrides | default({}) }}"
     blocksize: "1024k"
     ioengine: rbd
+    rw: write
     iodepth: 8
     numjobs: 1
-    run_bench: "{{ rbd_bench | default(True) }}"
+    run_bench: "{{ ebs_rbd_bench | default(false) }}"
   - src: "fio_direct_test.cfg.j2"
-    name: "fio_direct_read_test_RBD"
+    name: "fio_rbd_1M_read_test_EBS"
     override: "{{ fio_rbd_read_test_1024k_overrides | default({}) }}"
     blocksize: "1024k"
     ioengine: rbd
     rw: read
     iodepth: 8
     numjobs: 1
-    run_bench: "{{ rbd_bench | default(True) }}"
+    run_bench: "{{ ebs_rbd_bench | default(false) }}"
   - src: "fio_rw_mix.cfg.j2"
-    name: "fio_rw_mix_RBD"
+    name: "fio_rbd_1M_rw_mix_EBS"
     override: "{{ fio_rbd_rw_mix_1024k_overrides | default({}) }}"
     blocksize: "1024k"
     ioengine: rbd
     iodepth: 8
     numjobs: 1
-    run_bench: "{{ rbd_bench | default(True) }}"
+    run_bench: "{{ osa_rbd_bench | default(false) }}"
+  - src: "fio_direct_test.cfg.j2"
+    name: "fio_rbd_write_test_OSA"
+    override: "{{ fio_rbd_write_test_4k_overrides | default({}) }}"
+    blocksize: "4k"
+    ioengine: rbd
+    rw: write
+    iodepth: 8
+    numjobs: 1
+    run_bench: "{{ osa_rbd_bench | default(false) }}"
+  - src: "fio_direct_test.cfg.j2"
+    name: "fio_rbd_read_test_OSA"
+    override: "{{ fio_rbd_read_test_4k_overrides | default({}) }}"
+    blocksize: "4k"
+    ioengine: rbd
+    rw: read
+    iodepth: 8
+    numjobs: 1
+    run_bench: "{{ osa_rbd_bench | default(false) }}"
+  - src: "fio_rw_mix.cfg.j2"
+    name: "fio_rbd_rw_mix_test_OSA"
+    override: "{{ fio_rbd_rw_mix_4k_overrides | default({}) }}"
+    blocksize: "4k"
+    ioengine: rbd
+    iodepth: 8
+    numjobs: 1
+    run_bench: "{{ osa_rbd_bench | default(false) }}"
```

Comment:
should we default to true?
As it stands you have to set some vars in order for it to run any benchmarks at all - may be better to have an opt out instead.

Reply:
I'm fine with changing some or all of the defaults to true.  As more and more tests are added I didn't want the default to be to run all of the tests since there are so many now.  I think I would prefer to identify one set of tests, whether it is osa or rbd and only default that set to true.